R/qualification-ddi.R

```diff
@@ -290,7 +290,9 @@ generateDDIQualificationDDIPlot <- function(ddiPlotData, delta = 1) {
     color = "Caption",
     minRange = c(0.1, 10),
     residualsVsObserved = residualsVsObserved,
-    deltaGuest = delta
+    # Undefined delta in Configuration Plan corresponds to NULL
+    # In such case, default value should be 1 to prevent crash when defining data mapping
+    deltaGuest = delta %||% 1
```

Comment:
@pchelle The difference is when explicitly passing NULL as argument, right?
Before `generateDDIQualificationDDIPlot (ddiPlotData, NULL) `would result in `deltaGuest = NULL` and now it results in `deltaGuest = 1`, correct?

Reply:
This is correct.